openedx/core/djangoapps/commerce/__init__.py

```diff
@@ -0,0 +1,36 @@
+""" Commerce app. """
```

Comment:
Is the fact that we've got two Django apps named `commerce` going to cause trouble? You may also need to include this new app in the `INSTALLED_APPS` settings.

Though, actually, now that I think of it... Django 1.4 probably doesn't pick this up as an app at all, since it doesn't have a `models.py` file. So maybe this would be more appropriate to include in `openedx/core/lib`?